python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -5,7 +5,7 @@
     import unittest
```

Comment:
Ah, yes. GitHub has truncated the file and so it looked like we weren't. All is good.